charts/kube-prometheus-stack/templates/prometheus/rules-1.14/k8s.rules.yaml

```diff
@@ -33,26 +33,26 @@ spec:
       record: node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate
     - expr: |-
         container_memory_working_set_bytes{job="kubelet", metrics_path="/metrics/cadvisor", image!=""}
-        * on (namespace, pod) group_left(node) topk by(namespace, pod) (1,
-          max by(namespace, pod, node) (kube_pod_info{node!=""})
+        * on (cluster, namespace, pod) group_left(node) topk by(cluster, namespace, pod) (1,
```

Comment:
This was was automaticly changed by running the python script. No manual changes applied here.